cmake/msys/msys_helper.sh

```diff
@@ -90,33 +90,33 @@ if [ $? -ne 0 ]; then
 	if [ $? -ne 0 ]; then
         	err "ERROR: Could not build/install fltk -- Zyn needs this.  Exiting."
 	fi
-	
-#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe 	
+
+#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe
 	popd
 else
-	warn "  - Skipping, fluid binary already exists" 
+	warn "  - Skipping, fluid binary already exists"
 fi
 
 info "Downloading and building libogg $oggver"
 
-if [ ! -e $mingw_root/lib/libogg.dll.a ]; then 
-	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O $HOME/libogg-source.tar.xz
+if [ ! -e "$mingw_root/lib/libogg.dll.a" ]; then
+	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O "$HOME/libogg-source.tar.xz"
```

Comment:
That we won't set something like: `oggver="1 3 2"` in the future ? If this is what you're asking, the answer is: of course no.

If your question is that it knows that `oggver` is not an env var and is not modified by the script between the declaration and the use of the var etc. ; the answer is yes.

I plan to add shellcheck with astyle and kwstyle, so if a change like this is made, we will be warned.

But I don't see why we should quote this as it's currently not needed, probably won't (space in a version number in an url is not a common thing), makes the code harder to read and also because we'll be able to prevent this by adding shellcheck in the ci chain.